.gitignore

```diff
@@ -16,6 +16,8 @@
 *.swp
 *.user
 *.xml
+*.VC.opendb
```

Comment:
Would ideally be in it's own `PR`, though leave it for now. (Doesn't relate to the feature your adding).